### PR TITLE
tweak: Made header pill configurable

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: ad-hoc

Since we don't always want to show the OL pill in the header, it is now configurable. 

Screenshot 2023-05-08 at 10 36 35 AM

Screenshot 2023-05-08 at 10 36 25 AM

- [ ] Tests added?

### CODE REVIEW

File: test/screens/v2/candidate_generator/widgets/cr_departures_test.exs [hunk 1]
```diff
@@ -24,8 +24,7 @@ defmodule Screens.V2.CandidateGenerator.Widgets.CRDeparturesTest do
             direction_to_destination: 1,
             wayfinding_arrows: "right-down",
             priority: [1],
-            travel_time_to_destination: "9-12m",
-            show_via_headsigns_message: false
+            travel_time_to_destination: "9-12m"
```
Comment (hannahpurcell) [inline test/screens/v2/candidate_generator/widgets/cr_departures_test.exs:27]: Oh geez, this reminds me. I was supposed to update the audio to remove the travel time message. I forgot, that's my bad. Would you mind doing that, if you're not working on other things? [Slack link](https://mbta.slack.com/archives/C039ARUM48H/p1683124188886079)

Comment (cmaddox5) [reply test/screens/v2/candidate_generator/widgets/cr_departures_test.exs:27]: Yeah no problem. Changes have been pushed to this PR.

Comment (hannahpurcell) [reply test/screens/v2/candidate_generator/widgets/cr_departures_test.exs:27]: References to `travel_time_to_destination` can be removed like `show_via_headsigns_message`

**Review by jzimbel-mbta — APPROVED** [review]
LGTM! Checked with Hannah and she's good with the changes as well

**Pull request merged**